torchvision/prototype/models/resnet.py

```diff
@@ -231,11 +242,28 @@ class WideResNet101_2Weights(Weights):
     )
 
 
+# Maybe we should rename kwargs into model_kwargs to be more explicit? This is not a BC-breaking change I believe.
```

Comment:
It shouldn't be BC breaking you are right. But I think `kwargs` is more common idiom and renaming it might be confusing to users? Not sure.